### ISSUE TITLE
Guidelines for new conformance tests

### ISSUE THREAD

**Issue by garu** [twitter/twitter-text-conformance#26]
I'm writing a new twitter-text library and it already passes all tests in 'twitter-text-conformance', but I'm not really confident it's correct. I say that because I feel a lot of unit tests are missing.

Take the username validation in validate.yml, for example. It has a test that says "valid username: a-z < 20 characters" and another one saying "All numeric username are allowed". How about mix-and-match? Is 20 the overall username size limit? How about Unicode? is "@" a valid username? Should the unicode "＠" also be considered a valid username marker?

I had a lot of questions like that while I coded, and I still do. I'd love to volunteer and write all those tests, but I'm not the authority here so I can't pick what's valid and what's not off the top of my head - nor am I willing to try posts on my own Twitter account just for testing purposes (my followers would get crazy :)

tl;dr - Is there an implementation I can use as "correct"? This way I can use it as authoritative and see whether the new tests passes or fails.

Thanks!
